.qlty/qlty.toml

```diff
@@ -0,0 +1,100 @@
+# This file was automatically generated by `qlty init`.
+# You can modify it to suit your needs.
+# We recommend you to commit this file to your repository.
+#
+# This configuration is used by both Qlty CLI and Qlty Cloud.
+#
+#     Qlty CLI -- Code quality toolkit for developers
+#     Qlty Cloud -- Fully automated Code Health Platform
+#
+# Try Qlty Cloud: https://qlty.sh
+#
+# For a guide to configuration, visit https://qlty.sh/d/config
+# Or for a full reference, visit https://qlty.sh/d/qlty-toml
+config_version = "0"
+
+exclude_patterns = [
+  "*_min.*",
+  "*-min.*",
+  "*.min.*",
+  "**/.yarn/**",
+  "**/*.d.ts",
+  "**/assets/**",
+  "**/bower_components/**",
+  "**/build/**",
+  "**/cache/**",
+  "**/config/**",
+  "**/db/**",
+  "**/deps/**",
+  "**/dist/**",
+  "**/extern/**",
+  "**/external/**",
+  "**/generated/**",
+  "**/node_modules/**",
+  "**/seed/**",
+  "**/target/**",
+  "**/templates/**",
+  "**/testdata/**",
+  "**/vendor/**", "**/vendor/", "decidim-comments/app/assets/javascripts/decidim/comments/bundle.js", "decidim-comments/app/frontend/support/schema.ts", "node_modules/", "development_app/", "spec/decidim_dummy_app/", "coverage/", "webpack.report.html", "decidim_app-design/public/sw.js", "decidim_app-design/public/sw.js.map",
```

Comment:
Can you separate these in each line? if not adding new things here will make more difficult to work with git tooling

Reply:
Done in cfb28b3735